src/js/Controllers/UIController.js

```diff
@@ -193,6 +200,48 @@ class UIController {
                     this.onSystemContext("HMI_OBSCURED", context)
                 }
 
+                return null
+            case "SubtleAlert":
+
+                var tryAgainInfo = undefined;
+                var activeInteractionId = undefined;
+                const state3 = store.getState()
+                for (var appId in state3.ui) {
+                    var app = state3.ui[appId];
+                    if (app.isPerformingInteraction) {
+                        tryAgainInfo = 'A PerformInteraction is active';
+                        activeInteractionId = app.interactionId;
+                    } else if (app.alert.showAlert) {
+                        tryAgainInfo = (app.alert.subtle ? 'Another SubtleAlert' : 'An Alert') + ' is active';
+                        activeInteractionId = app.alert.msgID;
+                    }
+                }
+
+                if (activeInteractionId) {
+                    var tryAgainTime = this.endTimes[activeInteractionId] - Date.now();
+                    var rpc = RpcFactory.SubtleAlertErrorResponse(rpc.id, 4, tryAgainInfo);
```

Comment:
I can add enum definitions for those two enums, but I think that is outside the scope of this feature and will have to come later.

Reply:
Agreed, adding the result code enums in a js header is what we should do but it might be outside the scope of this feature. I think we’ll have to make a separate PR adding the enums and replacing all of the constants with the resultCodes.